deploy/scripts/build.js

```diff
@@ -5,7 +5,7 @@ const outbase = './src'
 const outdir = './dist'
 
 build({
-  entryPoints: glob.sync('src/*/*.ts'),
+  entryPoints: glob.sync(['src/*/*.ts', 'src/*/*/*.ts']),
```

Comment:
This is intentional to not include TypeScript utility files.

Rename the files to keep them in the team's folder
```diff
- deploy/scripts/src/cri-orange/address/test.ts
+ deploy/scripts/src/cri-orange/address.ts
```
```diff
- deploy/scripts/src/cri-orange/kbv/test.ts
+ deploy/scripts/src/cri-orange/kbv.ts
```

Reply:
Done